Jenkinsfile

```diff
@@ -0,0 +1,69 @@
+pipeline {
+  parameters {
+    choice(name: 'app', choices: ['showcase', 'cardio-patient', 'cardio'], description: 'Select the app to build and deploy')
```

Comment:
@amolht these app names do not match the app names in the workspace: "cardiovascular-health" and "cardiovascular-patient-app". I think the docker build command will fail for those apps.